### PR TITLE
Run tests in ci

### PR DESCRIPTION
**Pull request by bachstatter**

### CODE REVIEW

File: .github/workflows/run-tests.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+    branches: [master, dev]
```
Comment (artdgn) [inline .github/workflows/run-tests.yml:6]: `develop`?

**Review by artdgn — COMMENTED** [review]
wrong branch name?

File: .github/workflows/run-tests.yml (hunk 1 above)
Comment (bachstatter) [reply .github/workflows/run-tests.yml:6]: Nice catch.. Interesting that it ran either-way 🤔

Comment (bachstatter) [reply .github/workflows/run-tests.yml]: Pattern matching https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-branches

Comment (bachstatter) [reply .github/workflows/run-tests.yml]: Added an if statement to the deploy-keeper workflow to ensure we only deploy when on `develop` or `master`

**Review by artdgn — APPROVED** [review]

**Pull request merged**